### DISCUSSION TITLE
Do not reset filter options

### DISCUSSION THREAD

**Discussion by poznianski** [codewars/codewars.com#3059] [Ideas]
I use codewars on daily basis and its a bit frustrating every time set needed settings.
![image](https://github.com/codewars/codewars.com/assets/23197222/7d2f79b6-1bd7-48b5-a2db-c904ec93d251)
Would be extremely cool if you save it in local storage for example. And if I open site tomorrow settings will be the same as today.

**Comment by hobovsky** [comment]
Search parameters are a part of the URL. You can save a configuration of filters as a bookmark in your browser.
